### DISCUSSION TITLE
给hexo的配置增加一个默认夜间模式的选择

### DISCUSSION THREAD

**Discussion by liuliang520500** [tangly1024/NotionNext#2044] [Ideas]
**为什么提出这个新的特性改动**
我想给helo主题设置成默认为夜间模式，无从下手。WIDGET_DARK_MODE这个环境变量是设置是否显示夜间模式按钮，并不能设置默认为夜间模式

**描述一下你推荐的解决方案**
配置里面增加默认夜间模式

**Comment by tangly1024** [comment]
你好，可以在blog.config.js设置默认夜间模式：

https://github.com/tangly1024/NotionNext/blob/fc04903d45c0bb98e4aabc2c725c46f8f56b445e/blog.config.js#L11

当值为'dark'时网站默认为深色模式。
